src/audio/copier.c

```diff
@@ -719,6 +719,84 @@ static int copier_set_large_config(struct comp_dev *dev, uint32_t param_id,
 	}
 }
 
+static inline void divide_u64_to_u32s(uint64_t val, uint32_t *val_l, uint32_t *val_h)
```

Comment:
lets rename this since we are not doing a mathematical division.

Reply:
sure